src/main/java/uk/ac/ebi/eva/contigalias/controller/BaseController.java

```diff
@@ -31,6 +31,12 @@
 
     public static final int DEFAULT_PAGE_SIZE = 10;
 
+    public static final String API_PARAM_VALUE_PAGE_NUMBER = "Page number of required page. Page numbers start from 0" +
```

Comment:
this is actually not a page number, what about "PAGE_NUMBER_DESCRIPTION"? 
(I don't provide a code suggestion because it will be easier to rename in your IDE).

Also, "Page number of required page" is not really clear. What about "You  can provide a page index and page size to return only a subset of the data." and use that description for "size" too? maybe it makes sense to rename the parameter themselves because "size" is not obviously referring to the pages.